# Genomic-Systems
Application of Systems approaches to study the intersection of genomics and diseases

“Systems biology is the coordinated study of biological systems by (1) investigating the components of cellular networks and their interactions, (2) applying experimental high-throughput and whole-genome techniques, and (3) integrating computational methods with experimental efforts”. From [Klipp et al., 2006](https://books.google.com/books?hl=en&lr=&id=_5o5Eb9t3ucC&oi=fnd&pg=PR5&dq=Systems+Biology+in+Practice,+Wiley-VCH,+Berlin&ots=A5HumvjDfL&sig=DrkbtzDBWrJViMV4yAtmlo-ZfP4#v=onepage&q=Systems%20Biology%20in%20Practice%2C%20Wiley-VCH%2C%20Berlin&f=false). 

KEGG (Kyoto Encyclopedia of Genes and Genomes) is a "database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing.." From [KEGG Website](http://www.genome.jp/kegg/).

Grapov has developed a [teaching demo](https://github.com/dgrapov/TeachingDemos/blob/master/Demos/Pathway%20Analysis/KEGG%20Pathway%20Enrichment.md) for visualizing KEGG pathways. The author uses R packages pathview, KEGGREST, and KEGGGraph. A tutorial on KEGGREST is provided [here](https://bioconductor.org/packages/release/bioc/vignettes/KEGGREST/inst/doc/KEGGREST-vignette.html).

[Protege](https://protege.stanford.edu/) is a "free, open-source platform ... with a suite of tools to construct domain models and knowledge-based applications with ontologies." Belleau and Morissette have developed a [Kegg2RDF converter]( (written in Perl) as part of their [Pathway2GO](http://bio2go.sourceforge.net/poster%20Pathway2Go.pdf) project.  KEGG and Reactome are two key biological pathway resources. Reactome exists in [RDF](https://www.w3.org/RDF/) format; with KEGG now having this support. See also their [paper](https://www.sciencedirect.com/science/article/pii/S1532046408000415). RDF (Resource Description Framework) is used in semantic web applications for building models for data interchange. 

My eventual goal is to use all these resources to develop a systems level network for relating molecular pathways to diseases, as relevant to cardiovascular diseases. Cardiovascular disease is considered to be a complex disease since the ultimate impact is due to cumulative effect of multiple common risk alleles individually of small effect size rather than rare variants with large effects on disease risk. See [McPherson & Tybjaerg-Hansen  2016](https://www.ncbi.nlm.nih.gov/pubmed/26892958). Further, GWAS allows one to explore non-coding regions where most of these genetic loci are found. This requires backgrounds in computational science, data analytics, statistics, and cardiovascular pathology and pharmacology, and their application to GWAS. 
