Obesity is a complex disease, similar to cardiovascular, diabetic, and renal diseaases, in the sense that it is polygnic and genomic. Not all of the heredity is explained by the genes identified so far. It appears more research has been conducted in this area, perhaps because of the commercialization potential; much can be learned from here and adapted for other other diseases. 

Notes from "The Genetics of Obesity" by S.F.A. Grant , 2014, Springer, to be included here. I have handwritten notes to be typed in here for most of the chapters. 

I am directly entering notes from Chapter 7 here. Title: Functional Follow-Up of Genetic Variants using FTO as the Prime Example. Authors: S. Seehaus and U. Ruther, pp. 127

Summary: Obesity is the result of lifestyle and genetic factors. Tens of genes/ loci have been associated with obesity . *FTO* has shown the highest effect on body weight, the first found from GWAS studies, not uncovered earlier by single-gene functional genomics studies. This has been a highly active area of research over the past decade. *FTO*  is likely to act as a RNA demythelase, the biological pathway to higher body weight is not known. 

In mice, *Fto* was identified in 1999. It is one of the 6 genes affected by 'Fused toes' (*Ft*) mutation. Mice heterozygous for this 1.6 Mb deletion show forelimbs with fused toes and high thymic hyperplasia, likely caused through impaired apoptosis. Homozygous *Ft/Ft* embryos died between embryonic day 10.5 and 12.5 due to severe developmental malformation. *Fto* encodes a protein of 502 AA, 58 kDa, dissimilar to other known proteins. It is a large gene of 9 exons present in all vertebrates, and even in marine algae. When GWAS studies began, *FTO* was found to be correlated with BMI increase. Homozygous carriers weighed 3 Kg more and had a 1.67-fold higher risk for obesity. Most SNPs in the first intron are implicated. 

GWAS only indicate a loci and not a specific gene. Other possible genes in the loci are *RPGRIP1L*, which has a role in embryonic development. *Ex vivo* and in vitro analyses  (of subcutaneous fat of lean and obese women) showed *FTO* expression (but not *RPGRIP1L*) correlated positively with BMI. Both were downregulated with progression in adipocyte differentiation. *CUX1* may be a regulator of these two genes. Evidence is not strong. Current understanding also rules out RPGRIP1L.

FTO Expression profile in relation to nutritional states: Fto/FTO expression has been confirmed in the nucleus of various cell types and tissues. Highest levels are found in the brain, in hypothalamus and cerebellum. There is focus on FTO's role in regulating energy metabolism. Experiments altering nutritional states have limitations in controlling all the environmental conditions and  delivered mixed results. On the genetic side, FOXA2 seems to bind to the promoter of *FTO*, reducing the expression level of *FTO*. 

Functional studies in vitro: *Demythlase FTO*: *In silico* analysis ("performed on computer or via computer simulation") suggests *FTO* to be a member of the superfamily of non-heme dioxygenases. These are involved in posttranslational modifications, DNA repair and histone modifications, and act as cellular sensors for metabolism and oxygen. FTO seemed to demethylate 3-methylthymine in the ssDNA (single strand DNA) setting. It matched with parallel development that it may act as a RNA demethylase. This was verified with recombinant human and mouse FTO. FTO functioning as a DNA demethylase is unlikely.

The N terminus of FTO (32-326 AA) shows high sequence similarity t alkB and mammalian homologues, the C-terminal domain (327-498 AAs) shows no sequence similarity to any gene. N terminus is the catalytic domain (the extra loop prevents DNA activity). 

N6-methyladenosine (m<sup>6</sup>A) is another substrate for FTO in ssRNA. It is an abundant modification in mRNA.  In vitro, this methyltransferase complex preferentially methylates RNA oligonucleotides containing GGACU (from [Wiki](https://en.wikipedia.org/wiki/N6-Methyladenosine)). Overexpression of FTO led to decreased quantity of 
m<sup>6</sup>A in mRNA, whereas 90% knockdown of FTO led to an increase. This raised questions about m<sup>6</sup>A's role in biological processes. Mapping m<sup>6</sup>A has given ther human m<sup>6</sup>A RNA methylomes. 18,000 m6A sites in the transcripts of more than 7,000 human genes with a consensus sequence of [G/A/U] have been identified in human RNA. A group of RNA binding proteins ('m<sup>6</sup>a readers') specifically recognize the methylated adenosine on RNA.  Together with m6A methyltransferases (writers) and demethylases (erasers), these readers establish a complex mechanism of m<sup>6</sup>A regulation. Also, m<sup>6</sup)A modifications appeard to be **dynamically and reversibly regulated**.   

