Genomic Wide Association Study (GWAS), AKA Whole Genome association Study (WGAS), summarized from the Wiki reference [here](https://en.wikipedia.org/wiki/Genome-wide_association_study): It is an observational study of genetic variants across the whole genome. Statistical case-control methods are used to associate SNPs and (human) disease phenotypes/biomarkers.  

Sequencing the whole genome of each person in the study is expensive; so, only a subset is measured (using a custom genetic array chip): no CNV and only SNPs with enough variation in the reference population. A custom built **microarray** is then used to perform restricted genotyping for each person. Thus, GWAS does not directly genotype all variation in the genome. However, imputation of the genotypes to a reference panel, as from the HapMap project or the 1000 Genomes project, boosts the coverage of genomic variation. 

About 1 million SNPs are tabulated for comparison between the case and control groups. High Odds ratio (>1) with significant P-value (from a simple chi-squared test) is required to associate that SNP with a specific disease. [Wellcome 2009 GWAS study]() has these numbers for CAD (coronary artery disease): G-allele of SNP1 (*rs1333049*) was found in 52.6% of cases vs 44.6% of controls. Corresponding P-value was 5.0 x 10 <sup>-15</sup>. However, the G-allele of SNP2 (*

The following is extracted from a 2009 GWAS [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2719288/):  
Genetic variation may influence CAD risk both directly and indirectly (through other related risk phenotypes including hypertension, diabetes and hypercholesterolaemia). Only two loci have been identified from earlier Genome-wide linkage studies: ALOX5AP (arachidonate 5-lipoxygenase-activating protein) and LTA4H (leukotriene A4 hydrolase)). Genetic Association studies have identified several genetic variants affecting lipids, thrombosis, inflammation or vascular biology. This GWAS study, however, did  not support these findings.
The most notable new finding for CAD is the powerful association on chromosome 9p21.3
(Table 3; Fig. 5). Although the strongest signal is seen at rs1333049 (P=1.8×10-14),
associations are seen for SNPs across>100 kilobases. This region has not been highlighted in
previous studies of CAD or myocardial infarction40,43. The region of interest contains the
coding sequences of genes for two cyclin dependent kinase inhibitors, CDKN2A (encoding
p16INK4a) and CDKN2B (p15INK4b), although the most closely associated SNP is some
distance removed. Both genes have multiple isoforms, have an important role in the
regulation of the cell cycle and are widely expressed44, with CDKN2B known to be
expressed in the macrophages but not the smooth muscle cells of fibrofatty lesions45,46. It
is of interest that expression of CDKN2B is induced by transforming growth factor beta
(TGF-β) and that the TGF-β signalling system is implicated in the pathogenesis of human
atherosclerosis45,46. Besides CDKN2A and CDKN2B, the only other known gene nearby is
MTAP which encodes methylthioadenosine phosphorylase, an enzyme that contributes to
polyamine metabolism and is important for the salvage of both adenine and methionine.
MTAP is ubiquitously expressed, including in the cardiovascular system47. Further work is
required to determine whether the CAD association at this locus is mediated through
CDKN2A/B, MTAP or some other mechanism. The same region also shows replicated
evidence of association to T2D in the WTCCC and other data sets19,21,22, though different
SNPs seem to be involved.
None of the loci showing more modest associations with CAD (Table 4) includes genes
hitherto strongly implicated in the pathogenesis of CAD. A potentially interesting
association is at rs6922269 (P=6.3×10-6), an intronic SNP in MTHFD1L, which encodes
methylenetetrahydrofolate dehydrogenase (NADP+-dependent) 1-like, the mitochondrial
Page 9
Nature. Author manuscript; available in PMC 2009 July 31.
Europe PMC Funders Author Manuscripts Europe PMC Funders Author Manuscripts
isozyme of C1-tetrahydrofolate (THF) synthase48,49. C1-THF synthases interconvert the
one carbon units carried by the biologically active form of folic acid, C1-tetrahydrofolate.
These are used in a variety of cellular processes including purine and methionine
synthesis48. Another enzyme in the same pathway, methylene THF reductase (encoded by
MTHFR) is subject to a common mutation which influences plasma homocysteine level50
and has been associated with increased risk of coronary and other atherosclerotic disease51.
The possibility of a link between variants in MTHFD1L and CAD risk is supported by
evidence that MTHFD1L activity also contributes to plasma homocysteine52 and that
defects in the MTHFD1L pathway may increase plasma homocysteine level48,53.
An intronic SNP in ADAMTS17 (a disintegrin and metalloproteinase with thrombospondin
motifs 17), which showed modest association (rs1994016; P=1.1×10-4) in our primary
analysis, showed a much stronger association in the expanded reference group analysis (see
below and Supplementary Table 9). Although the specific function of ADAMTS17 has not
been determined, other members of the ADAMTS family have been implicated in vascular
extracellular matrix degradation, vascular remodelling and atherosclerosis54,55.








Related terms:
* Haplotype - a group of alleles inherited together from a single parent. They are conserved across many generations. 1000 Genomes Project is an example; it also means a set of linked SNP alleles that occur together (at a high statistical significance). The HapMap project is using this for links to diseases; genetic test companies use this term to refer to specific mutations (e.g., STR mutation). 
* Imputation - statistical inference of unobserved genotypes. Starting with known haplotypes in a population (as with HapMap or the 1000 Genomes project), genetic variants are statistically associated with a phenotype. GWAS uses imputation to increase the coverage of SNPs, beyond those included in the microarray. Also, imputation allows overlap and meta-analysis of datasets obtained from different arrays. 
